src/binaryapi/TradingTimes.js

```diff
@@ -28,7 +28,7 @@ class TradingTimes {
                 let nextUpdate = this._nextUpdateDate();
 
                 if (!nextUpdate) {
-                    const now = new Date();
+                    const now = this._serverTime.getLocalDate();
                     const getUpdateDate = () => new Date(`${this.lastUpdateDate}T00:00:00Z`);
```

Comment:
I suspect there is a bug in this line of code. I've set it to use UTC time but then calculate the difference using local time. Changing everything to use UTC time via `ServerTime` should resolve this issue.

Reply:
@brucebinary yup. youre right. it must be in utc